### DISCUSSION TITLE
怎么丢弃resp_ip中指定的IP

### DISCUSSION THREAD

**Discussion by likevi9528** [IrineSistiana/mosdns#780] [Q&A]
现有一个域名，经过国内外DNS解析会获得三个返回IP，但是其中有一个在国内访问不了(通过itdog和本地测出来)。有没有什么方法可以去掉这个IP。
我现在只能使用hosts解决这个问题，有没有什么方法只需要更新blackIpList，就会自动在resp_ip中去除blackIpList中的IP。
我的想法是下面的方式，但会导致返回空的IP。

  - tag: black_ipList
    type: "ip_set"
    args:
      files:
        - "./rules/custom_black_ipList.txt"
  
  - tag: world_resolve
    type: "sequence"
    args:
      - exec: $world_DNS
   **- matches: "resp_ip $black_ipList"
        exec: drop_resp**
      - exec: jump has_resp_sequence
      - exec: query_summary world_DNS

**Comment by smallst** [comment]
drop_resp 确实会返回空，你这个需求听起来需要重新解析，所以你应该jump 到 一个新的解析sequence上
